pkg/apis/servicecatalog/validation/broker.go

```diff
@@ -98,13 +99,50 @@ func validateServiceBrokerSpec(spec *sc.ServiceBrokerSpec, fldPath *field.Path)
 				field.Required(fldPath.Child("authInfo"), "auth config is required"),
 			)
 		}
-
 	}
 
 	if spec.InsecureSkipTLSVerify && len(spec.CABundle) > 0 {
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("caBundle"), spec.CABundle, "caBundle cannot be used when insecureSkipTLSVerify is true"))
 	}
 
+	if "" == spec.RelistBehavior {
+		allErrs = append(allErrs,
+			field.Required(fldPath.Child("relistBehavior"),
+				"brokers must have a relistBehavior"))
+	}
+
+	isValidRelistBehavior := spec.RelistBehavior == "Duration" ||
+		spec.RelistBehavior == "Manual"
+	if !isValidRelistBehavior {
+		errMsg := fmt.Sprintf("unknown relistBehavior found: %v", spec.RelistBehavior)
```

Comment:
"relist behavior must be "Manual" or "Duration""

Reply:
Or use field.NotSupported which takes a []string of valid values.

Reply:
Or use field.NotSupported which takes a []string of valid values.